script/ci

```diff
@@ -2,6 +2,9 @@
 
 set -e
 
+. script/lib.sh
+
 script/fetch-fixtures
 script/check-mallocs
-script/test
+scan_build make -j2
+script/test -b
```

Comment:
Ah, this is a remnant of an older version of this patch. I will remove that `scan_build make -j2` step